test_lib.go

```diff
@@ -2,44 +2,81 @@ package main
 
 import (
 	"os"
+	"reflect"
+	"testing"
 
 	"github.com/flow-hydraulics/flow-pds/service/app"
 	"github.com/flow-hydraulics/flow-pds/service/common"
 	"github.com/flow-hydraulics/flow-pds/service/config"
 	"github.com/flow-hydraulics/flow-pds/service/http"
+	"github.com/onflow/flow-go-sdk/client"
+	"google.golang.org/grpc"
 )
 
 func getTestCfg() *config.Config {
-	return &config.Config{
-		DatabaseDSN:  "test.db",
-		DatabaseType: "sqlite",
+	cfg, err := config.ParseConfig(&config.ConfigOptions{EnvFilePath: ".env.test"})
+	if err != nil {
+		panic(err)
 	}
+
+	cfg.DatabaseDSN = "test.db"
+	cfg.DatabaseType = "sqlite"
+
+	return cfg
 }
 
-func getTestApp(cfg *config.Config) (*app.App, func()) {
-	db, err := app.NewGormDB(cfg)
+func getTestApp(cfg *config.Config, poll bool) (*app.App, func()) {
+	flowClient, err := client.New(cfg.AccessAPIHost, grpc.WithInsecure())
+	if err != nil {
+		panic(err)
+	}
+
+	db, err := common.NewGormDB(cfg)
 	if err != nil {
 		panic(err)
 	}
-	store := app.NewGormStore(db)
+
+	// Migrate app database
+	if err := app.Migrate(db); err != nil {
+		panic(err)
+	}
+
 	clean := func() {
-		os.Remove(cfg.DatabaseDSN)
+		if cfg.DatabaseType == "sqlite" {
+			os.Remove(cfg.DatabaseDSN)
```

Comment:
Unhandled error, a warning message is probably enough in this case.

Reply:
Good catch! I will incorporate this in the next PR.